spyder/plugins/editor/widgets/tests/test_goto.py

```diff
@@ -140,3 +140,17 @@ def interact():
     code_editor.filename = None
     code_editor._last_hover_pattern_key = None
     code_editor._last_hover_pattern_text = None
+
+
+def test_pattern_highlight_regression(qtbot, editorbot):
+    """Fix regression on spyder-ide/spyder#11376. """
```

Comment:
There's a blank space at the end of this docstring,

Reply:
Ok, let's forget about this one.

Reply:
I missed this comment 😆

Reply:
No biggie.

Reply:
No biggie.